src/Compilers/Core/Portable/Operations/ControlFlowBranch.cs

```diff
@@ -28,16 +29,29 @@ public sealed class ControlFlowBranch
             IsConditionalSuccessor = isConditionalSuccessor;
         }
 
+        /// <summary>
+        /// Source basic block of this branch.
+        /// </summary>
         public BasicBlock Source { get; }
 
+        /// <summary>
+        /// Destination basic block of this branch.
+        /// </summary>
         public BasicBlock Destination { get; }
 
+        /// <summary>
+        /// Semantics associated with this branch.
+        /// </summary>
         public ControlFlowBranchSemantics Semantics { get; }
 
+        /// <summary>
+        /// Indicates if this branch represents <see cref="BasicBlock.ConditionalSuccessor"/> of the <see cref="Source"/> basic block.
+        /// </summary>
         public bool IsConditionalSuccessor { get; }
 
         /// <summary>
-        /// What regions are exited (from inner most to outer most) if this branch is taken.
+        /// Regions exited if this branch is taken.
+        /// Ordered from the innermost region to the outermost region.
         /// </summary>
         public ImmutableArray<ControlFlowRegion> LeavingRegions
```

Comment:
>LeavingRegions [](start = 49, length = 14)

Never returns `default`.

Reply:
I believe that is an implicit contract for all of our public APIs returning `ImmutableArray`. I can still add it as a remark if you feel strongly about it.

---
In reply to: [193192712](https://github.com/dotnet/roslyn/pull/27477#discussion_r193192712) [](ancestors = 193192712)